law.cfg

```
@@ -8,7 +8,7 @@ inherit: $CF_BASE/law.cfg
 
 columnflow.tasks.cms.inference
 columnflow.tasks.cms.external
-dijet.tasks
+dijet.tasks.{jersf}
```

Since it's just one module, the braces aren't necessary.
```suggestion
dijet.tasks.jersf
```